rubrics/impl/src/main/java/org/sakaiproject/rubrics/logic/RubricsServiceImpl.java

```diff
@@ -761,7 +761,49 @@ private String putRubricResource(String targetUri,String json, String toolId) th
         }
     }
 
-    //TODO generate a public String deleteRubricAssociation(String tool, String id)
+    /**
+     * Delete all the rubric associations for a samigo assessment.
+     * @param itemId The formatted assessment id.
+     */
+	public void deleteSamigoAssociatedResources(String itemId) {
```

Comment:
So this method should generalized, do we expect to create similar methods for every tool?